SSA/Projects/InstCombine/Base.lean

```diff
@@ -62,7 +62,7 @@ def BitVec.width {n : Nat} (_ : BitVec n) : Nat := n
 
 instance : TyDenote Ty where
 toType := fun
-  | .bitvec w => Option <| BitVec w
+  | .bitvec w => Option (BitVec w)
```

Comment:
I guess this is a preference thing, but usually avoiding parenthesis scales better for readability (I think that's the Haskell style at least)

Reply:
I dropped this change as it was unrelated.